### PR TITLE
Add SoundSwitch v5.11.2

### PR DESCRIPTION
**Pull request by Belphemur**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/22331)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Hmm, can you convert this to a multi manifest?
Then we can do
```
Installers:
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.1/SoundSwitch_v5.11.1.18775_Release_Installer.exe
  InstallerType: inno
  InstallerSha256: cd5eecdd564822fc6d385fd396bb88f850c90971a600bc6e2224c5973a906e42
  InstallerSwitches:
  Custom: /CLOSEAPPLICATIONS /CURRENTUSER
- Architecture: x64
  Scope: machine
  InstallerUrl: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.1/SoundSwitch_v5.11.1.18775_Release_Installer.exe
  InstallerType: inno
  InstallerSha256: cd5eecdd564822fc6d385fd396bb88f850c90971a600bc6e2224c5973a906e42
  InstallerSwitches:
  Custom: /CLOSEAPPLICATIONS /ALLUSERS
```
for update compatibility since it recommends machine wide.

Also the version is 5.11.1.18775

**Review by Belphemur — COMMENTED** [review]
@OfficialEsco Done also moving to the latest version directly.

**Comment by Belphemur** [comment]
I'm using Cloudflare and I have the feeling the bot got blocked by it and can't do the URL validation ...

Is there a way to skip that ?

**Comment by OfficialEsco** [comment]
```
-----------------------------------------------------
Manifest: manifests\a\AntoineAflalo\SoundSwitch\5.11.2.39643
-----------------------------------------------------
URI: https://www.aaflalo.me/, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://soundswitch.aaflalo.me/, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://github.com/Belphemur/SoundSwitch/blob/dev/LICENSE.txt, Validation result: Passed
URI: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.2/SoundSwitch_v5.11.2.39643_Release_Installer.exe, Validation result: Passed
URI: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.2/SoundSwitch_v5.11.2.39643_Release_Installer.exe, Validation result: Passed
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/22331#issuecomment-887234266

File: manifests/a/AntoineAflalo/SoundSwitch/5.11.2.39643/AntoineAflalo.SoundSwitch.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: AntoineAflalo.SoundSwitch
+PackageVersion: 5.11.2.39643
+PackageLocale: en-US
+Publisher: Antoine Aflalo
+PublisherUrl: https://www.aaflalo.me/
+Author: Antoine Aflalo
+PackageName: SoundSwitch
+PackageUrl: https://soundswitch.aaflalo.me/
```
Comment (OfficialEsco) [inline manifests/a/AntoineAflalo/SoundSwitch/5.11.2.39643/AntoineAflalo.SoundSwitch.locale.en-US.yaml:11]:
```suggestion
#PublisherUrl: https://www.aaflalo.me/
Author: Antoine Aflalo
PackageName: SoundSwitch
#PackageUrl: https://soundswitch.aaflalo.me/
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Not sure why some sites do this..
```
URI: https://www.aaflalo.me/, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://soundswitch.aaflalo.me/, Validation result: Failed, Http status code: ServiceUnavailable 
URI: https://github.com/Belphemur/SoundSwitch/blob/dev/LICENSE.txt, Validation result: Passed
URI: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.2/SoundSwitch_v5.11.2.39643_Release_Installer.exe, Validation result: Passed
URI: https://github.com/Belphemur/SoundSwitch/releases/download/v5.11.2/SoundSwitch_v5.11.2.39643_Release_Installer.exe, Validation result: Passed
```

**Comment by Belphemur** [comment]
I'm the developer of SoundSwitch and I also own both website that the bot can't validate.

You can check manually that both website are working:
* https://www.aaflalo.me
* https://soundswitch.aaflalo.me

Like I said both of those are behind CloudFlare and I'm pretty sure the validation url pipeline was detected as a bot and blocked by CloudFlare.

Doest anybody have the IP of the pipeline that I can whitelist ?

**Comment by OfficialEsco** [comment]
Ahhhh i see, not familiar with CloudFlare hosting, one of these guys can probably answer that question https://github.com/microsoft/winget-pkgs/issues/15674#issue-908889039
However our main man @denelon is away and the rest don't like to respond 🤔
We can try to ping @KevinLaMS, @ChuckFerring, @jamespik and @palenshus which will be online in a few hours.

**Comment by palenshus** [comment]
Sorry for the delay, we'll discuss this and get back soon

**Comment by Belphemur** [comment]
@palenshus I've fixed the issue related to the URL validation, I've whitelisted the ASN of the Azure pipeline to be sure it won't cause an issue in the future.

The default bot protection of CloudFlare was blocking the pipeline.

**Review by OfficialEsco — APPROVED** [review]

**Comment by palenshus** [comment]
Nice, glad you were able to get it through @Belphemur! I'm not sure if the IP is static and will always stay the same, so hopefully you don't hit this again!

**Comment by denelon** [comment]
@OfficialEsco & @Belphemur I'll see if there is any way we can detect CloudFlare hosting. I've used their service on some other projects, but I wasn't aware of the potential for blocking.

**Comment by Belphemur** [comment]
@denelon It's related to their new Bot detection feature that I had enabled on my website.

I imagine, it relies on some kind of heuristic that detected the multiple call to validate the website when I was pushing on the branch.

I had to disable the bot detection because the firewall override didn't work for that feature.

**Pull request merged**